### DISCUSSION TITLE
240314

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#42] [팀회의]
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
ui아키택쳐 공부..?
## 오늘 할 일
로그인 열리면 axios연동, 내일까지 recat-hook-form 변경
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
x

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
캘린더 UI 작업, 
캘린더 context로 변경,
캘린더 주간 월간 합쳐서 mode로 prop 설정 
## 오늘 할 일
메인페이지 UI 수정, 캘린더 페이지 캘린더 수정
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 코드 리뷰가 좀 더 활성화되었으면 좋겠습니다 ㅎㅎ

## 어제 한 일
- 사이드바 관련 UI (툴팁, 드롭다운)
- 사이드바 pathname 받아서 현재 페이지 버튼에 포커스 되도록

## 오늘 할 일
- 색상 변수 적용
- 페이지 레이아웃 사이즈 수정
- 그룹 페이지 만들기
- 칸반보드 페이지 만들기

## 오늘의 일정 (코어 타임)
-  x

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
UI 완성
## 오늘 할 일
포트 열리는대로 유저기능 구현
## 오늘의 일정 (코어 타임)
2시부터 일정이 있어 참여 못합니다 저녁 때 분발하겠습니다 논의사항 남겨주시면 5시쯤부터 확인 가능 할 것 같습니다.
## 멘토님 질문 사항
